infra/app/service/main.tf

```diff
@@ -35,6 +35,8 @@ locals {
 
   service_name = "${local.prefix}${module.app_config.app_name}-${var.environment_name}"
 
+  bucket_name = "${local.prefix}${module.project_config.project_name}-${local.service_name}-storage"
```

Comment:
Is `-storage` redundant for an S3 bucket name?

Reply:
hm, good point. will remove.